dotCMS/src/main/webapp/html/portlet/ext/contentlet/view_bulk_actions_inc.jsp

```diff
@@ -357,8 +357,8 @@
     }
 
```

Comment:
instead put these two lines insted line 344 

`const entity = data ? data.entity : null; bulkWorkflowActionCallback(entity);`

this because the angular callback use the function `bulkWorkflowActionCallback ` and come without entity.